OsmAnd/src/net/osmand/plus/FavouritesDbHelper.java

```diff
@@ -598,6 +752,12 @@ public int compare(FavouritePoint o1, FavouritePoint o2) {
 				if (res == 0) {
 					res = collator.compare(s1, s2);
 				}
+				if (s1.equals(WORK) && s2.equals(PARKING) && o1.getCategory().equals(PERSONAL) && o2.getCategory().equals(PERSONAL)) {
```

Comment:
Something wrong with sorting. Home absent and it is not translation dependent.

Reply:
It's hardcoded string HOME always first

Reply:
On the UI we need to operate with translations and not with constants.